packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -160,20 +158,22 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = {
             },
             "& > svg": {
                 fill: darkModeNeutralForegroundRest,
-                [highContrastSelector]: {
-                    fill: "ButtonText",
-                },
+                ...highContrastForeground,
+            },
+            [highContrastSelector]: {
+                background: "none",
             },
         },
         "& $carousel_sequenceIndicator": {
             "&::before": {
                 background: darkModeNeutralFillStealthRest,
                 "border-color": darkModeNeutralOutlineRest,
+                ...highContrastButtonColorIndicator,
             },
             "&$carousel_sequenceIndicator__active": {
                 "&::before": {
                     background: darkModeNeutralFillStealthRest,
-                    ...highContrastHighlightBackground,
+                    ...highContrastHighlightColorIndicator,
                 },
             },
```

Comment:
This could also be consolidated a little,

```typescript
"&::before": {
    "border-color": darkModeNeutralOutlineRest,
},
"&::before, &$carousel_sequenceIndicator__active": {
    background: darkModeNeutralFillStealthRest,
    ...highContrastHighlightColorIndicator,
}
```